dts/arm/nxp/nxp_rt1160_cm4.dtsi

```diff
@@ -60,10 +60,10 @@
 			nxp,mem2mem;
 			nxp,a_on;
 			reg = <0x40c14000 0x4000>,
-				<0x400c1800 0x4000>;
+				<0x40c18000 0x4000>;
 			clocks = <&ccm IMX_CCM_EDMA_CLK 0x7C 0x000000C0>;
 			status = "disabled";
-			label = "EDMA1";
+			label = "DMA_1";
```

Comment:
this coould be DMA_0, as for m4 core this is the only dma, which will aligned with test application default setttings